### PR TITLE
Private NAT Gateway

### PR DESCRIPTION
**Pull request by aguillenv**
Part of #844

### CODE REVIEW

File: src/modules/0.0.9/aws_vpc/index.ts
Comment (dfellis) [inline src/modules/0.0.9/aws_vpc/index.ts:218]: Do we want to hold this PR until the EIP PR is landed, or would you rather do a follow-up PR for this?

Comment (aguillenv) [reply src/modules/0.0.9/aws_vpc/index.ts:218]: I'd like to do it in a follow up task after this one and #858 are merged

**Review by depombo — APPROVED** [review]

**Pull request merged**
